rocAL_pybind/amd/rocal/readers.py

```diff
@@ -350,3 +350,15 @@ def mxnet(path, stick_to_shard=False, pad_last_batch=False):
     mxnet_metadata = b.mxnetReader(
         Pipeline._current_pipeline._handle, *(kwargs_pybind.values()))
     return mxnet_metadata
+
+
+def numpy(*inputs, file_root='', num_shards=1,
+          random_shuffle=False, shard_id=0, stick_to_shard=False, pad_last_batch=False):
```

Comment:
Please remove 
          random_shuffle=False, shard_id=0, stick_to_shard=False, pad_last_batch=False):
to prev line